db/migration.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
maybe name this file `models.py` `migration.py/migrations.py` has another meaning in web development